src/composables/multisigTransactions.ts

```diff
@@ -12,6 +12,7 @@ import {
 } from '../popup/utils';
 import { useSdk } from './sdk';
 import { useMultisigAccounts } from './multisigAccounts';
+import { useTopHeaderData } from './topHeader';
```

Comment:
`topHeader` term is a bit confusing as it has a meaning in the UX space.

Reply:
This composable is written around this call `sdk.api.getTopHeader()`. But I'm open for suggestions about better name. Can you suggest one?

Reply:
Ideally, it could be renamed to `useHeadBlock` or `useTopBlock`. but renaming this depends on https://github.com/aeternity/superhero-wallet/pull/1890#discussion_r1124096202.

Right now it looks fine as it is a wrapper around `sdk.api.getTopHeader()`